app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -26,6 +31,25 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope',
     miqService.sparkleOff();
   }
 
+  $scope.storageManagerChanged = function (id) {
+    miqService.sparkleOn();
+    
+    $http.get('/cloud_volume/storage_manager_form_fields/' + id)
```

Comment:
Should I look for this now or is it planned for future releases? That would complicate things a bit and I think definitely require multiple calls. Is there perhaps an example of this approach already somewhere in the code?